echopype/tests/test_process_ad2cp.py

```diff
@@ -37,3 +38,111 @@ def test_process():
             assert testing_group.equals(base_group), f"process ad2cp failed on {file} with group {group}"
 
         os.remove(tmp_file)
+
+OCEAN_CONTOUR_UNPARSED = [
+    "./echopype/test_data/ad2cp/SG198_0097_a.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0097_b.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0099_a.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0099_b.ad2cp"
+]
+
+OCEAN_CONTOUR_PARSED = [
+    "./echopype/test_data/ad2cp/SG198_0097_a.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0097_b.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0099_a.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0099_b.ad2cp.nc"
+]
```

Comment:
Could you change:
- `OCEAN_CONTOUR_UNPARSED` to `AD2CP_UNPARSED`, and 
- `OCEAN_CONTOUR_PARSED` to `AD2CP_PARSED_BY_OCEAN_CONTOUR`?
This way we are explicit about what these refer to.